engine/src/main/java/org/hibernate/validator/internal/util/ReflectionHelper.java

```diff
@@ -253,6 +253,28 @@ public static Object getValue(Method method, Object object) {
 	}
 
 	/**
+	 * Indicates if the type is considered indexable (ie is an <code>Iterable</code>, an array or a <code>Map</code>).
```

Comment:
Could you use `{@code ...}` over `<code>...</code>` (as per http://hibernate.org/validator/contribute/)? If it's the other style still in some places, that's left-overs from earlier times (fix stuff you see as you go, if you like).